src/script/entity/User/User.ts

```diff
@@ -210,10 +212,14 @@ export class User {
       return this.devices().every(client_et => client_et.meta.isVerified?.());
     });
     this.isMLSVerified = ko.pureComputed(() => {
-      if (this.devices().length === 0 && !this.isMe) {
-        return false;
+      if (this.devices().length === 0) {
+        if (!this.isMe) {
+          return false;
+        }
+        const clientState = container.resolve(ClientState);
```

Comment:
Hmm it seems wrong to me to have ClientState resolved on User's entity level 🤔 How about just adding a field to user entity next to `devices` called e.g. `currentDevice` and set it once the self client is initialised?

Reply:
I thought about that, but threw it in the bin since this code is only used in one place and the user doesn't take use of the selfClient in any other way.

We can change that if it is necessary.

Reply:
That would be nice, User entity should be as dumb as possible

Reply:
oh yeah, entities using the app state is definitely something that, architecture-wise, might create tough to debug situation. 
Explicitely setting the `selfDevice` at load time feels safer and cleaner in term of app-layering

Reply:
Since the function returned an observable, the localClient in User is also an Observable. Doesn't hurt :)